src/Symfony/Component/Asset/VersionStrategy/JsonManifestVersionStrategy.php

```diff
@@ -63,6 +63,6 @@ private function getManifestPath(string $path): ?string
             }
         }
 
-        return isset($this->manifestData[$path]) ? $this->manifestData[$path] : null;
+        return $this->manifestData[$path] ?? null;
```

Comment:
Please revert all changes that are not needed for the PR to be complete.
Please submit them as a separate PR on a lower branch if you're up to (this one on 4.4, the other on 3.4 I suppose?)

Reply:
Of course, I didn't think to the maintainability issues implies by this change on master.

Reply:
Reverted.

PR done for assertions in 3.4 (https://github.com/symfony/symfony/pull/35793).
Regarding your comment https://github.com/symfony/symfony/pull/33225#issuecomment-522338660, I'm not sure if I should submit a PR to convert `isset` to null coalescing.